app/assets/javascripts/fe/fe.common.js.erb

```diff
@@ -57,12 +57,12 @@ function setUpSortables() {
 }
 
 function setUpCalendars() {
-	now = new Date();
-	year = now.getFullYear() + 10;
-	$('[data-calendar]').datepicker({
+  now = new Date();
+  year = now.getFullYear() + 10;
+  $('[data-calendar]').datepicker({
     changeYear:true,
-		yearRange: '1950:' + year,
-    dateFormat: 'yy-mm-dd'
```

Comment:
Just wanted to check that you intended to change the default date format as well as per https://github.com/CruGlobal/fe/pull/63/files#diff-e54dd103df675ce9f0b934e94ff0ca8bR28 ?